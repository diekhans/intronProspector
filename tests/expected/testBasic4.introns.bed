chr10	5453560	5454264	sj10	30	+
chr10	5453560	5454264	sj11	11	-
chr10	5454522	5454947	sj12	32	+
chr10	5454522	5454947	sj13	23	-
chr10	5455118	5456086	sj14	23	+
chr10	5455118	5456086	sj15	22	-
chr10	5456273	5456587	sj16	23	+
chr10	5456273	5456587	sj17	23	-
