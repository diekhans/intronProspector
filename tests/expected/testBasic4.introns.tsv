chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang	confidence
chr10	5453560	5454264	+	30	0	0	68	81	3.67
chr10	5453560	5454264	-	11	0	0	59	86	2.55
chr10	5454522	5454947	+	30	2	0	87	91	4.24
chr10	5454522	5454947	-	23	0	0	77	74	2.91
chr10	5455118	5456086	+	23	0	0	85	87	4.14
chr10	5455118	5456086	-	22	0	0	85	90	3.79
chr10	5456273	5456587	+	23	0	0	86	79	3.97
chr10	5456273	5456587	-	23	0	0	78	78	4.02
